go/encoding/basex/bases.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package basex
+
+const skipChars = "\t\n\r !\"#$%&'()*+,-./0:;<=>?@IOl[\\]^_`{|}~"
+const base58EncodeStd = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz"
+
+// Base58StdEncoding is the standard base58-encoding, with Strict mode enforcing
+// no foreign characters
+var Base58StdEncodingStrict = NewEncoding(base58EncodeStd, 19, "")
+
+// Base58StdEncoding is the standard base58-encoding. Foreign characters are ignored
+// as long as they're from the blessed set.
+var Base58StdEncoding = NewEncoding(base58EncodeStd, 19, skipChars)
+
+const base62EncodeStd = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
```

Comment:
This is ASCII order, but it's not the order Base64 uses. Which one should we go with?